ltiauthenticator/lti13/auth.py

```diff
@@ -119,11 +119,20 @@ class LTI13Authenticator(OAuthenticator):
     def login_url(self, base_url):
         return url_path_join(base_url, "lti13", "oauth_login")
 
+    def callback_url(self, base_url):
+        return url_path_join(base_url, "lti13", "oauth_callback")
+
+    def config_url(self, base_url):
+        return url_path_join(base_url, "lti13", "config")
+
+    def jwks_url(self, base_url):
+        return url_path_join(base_url, "lti13", "jwks")
```

Comment:
Are these provided for consistency with existing code base or something else?

Reply:
These are provided for consistency, i.e. to have a single point of truth for the paths to the handler. They are also available to the handlers via the `authenticator` attribute. So we do not need to guess the paths.

Reply:
:+1: okay lets do it!